src/clr-addons/notification/notification.scss

```diff
@@ -78,3 +80,10 @@
     }
   }
 }
+
+@media screen and (max-width: map-get($clr-grid-breakpoints, sm)) {
```

Comment:
why dont add the media query inside the already existing selector for the wrapper? see line 18